src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml

```diff
@@ -284,12 +289,14 @@
                         </Button>
 
                         <!-- edit/properties button -->
-                        <Button x:Name="ButtonEdit" Grid.Row="0" Grid.Column="4" Style="{StaticResource EditButtonStyle}">
+                        <Button x:Name="ButtonEdit" Grid.Row="0" Grid.Column="4" Style="{StaticResource EditButtonStyle}"
+                                AutomationProperties.Name="{x:Static components:Resources.ConnectionManager_EditConnection}">
```

Comment:
These buttons (`ButtonEdit` and `ButtonDelete`) have automation name specified in their styles already